### PR TITLE
PIM-4460, PIM-4461, PIM-4462 and PIM-4489

### PR DESCRIPTION
**Pull request by sumbobyboys**
| Q                    | A
| -------------------- | ---
| Bug fix?             | Y
| New feature?          | N
| BC breaks?           | N
| CI currently passes? | Y
| Tests pass?          | Y
| Scenarios pass?      | ?
| Checkstyle issues?*  | N
| PMD issues?**        | N
| Changelog updated?   | N
| Fixed tickets        | PIM-4460, PIM-4461, PIM-4462 and PIM-4489
| DB schema updated?   |
| Migration script?    |
| Doc PR               |

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/panel/completeness.js [hunk 1]
```diff
@@ -23,9 +23,12 @@ define(
             className: 'panel-pane',
             code: 'completeness',
             events: {
-                'click header': 'switchChannel',
+                'click header': 'switchLocale',
                 'click .missing-attributes span': 'showAttribute'
             },
+            /**
```
Comment (solivier) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/panel/completeness.js]: no spaces before ?

File: src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.en.yml [hunk 2]
```diff
@@ -123,7 +123,7 @@ pim_enrich:
                 disable: Disable
                 disabled: Disabled
                 download_pdf: PDF
-                copy: Copy
+                copy: Compare / Translate
```
Comment (nidup) [inline src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.en.yml]: change the key?

File: src/Pim/Bundle/UIBundle/Resources/public/css/pim.less [hunk 3]
```diff
@@ -907,9 +907,11 @@ h3.tab-header {
     .flex(0);
   }
 
-  & > .panels > .panel-container {
-    width: auto;
+  & > .panels {
     .flex(1);
```
Comment (nidup) [inline src/Pim/Bundle/UIBundle/Resources/public/css/pim.less]: flex !

**Comment by nidup** [comment]
gtm

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/panel/completeness.js (hunk 1 above)
Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/form/panel/completeness.js]: I prefer with a blank line too.

**Comment by phaseinducer** [comment]
GTM

**Comment by jmleroux** [comment]
GTM

**Pull request merged**